From 86d4feee0b9887eb9cdf9a2432bc1f48a6d98658 Mon Sep 17 00:00:00 2001
From: Meng Li <Meng.Li@windriver.com>
Date: Tue, 16 May 2023 02:10:18 +0000
Subject: [PATCH] arm64: zynqmp: enable zcu102 rev1.0 board by default for
 secure boot

Because Xilinx zynqmp secure boot is developed based on zcu102
rev1.0 board, enable zcu102 rev1.0 board by default for secure boot.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 configs/xilinx_zynqmp_virt_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/xilinx_zynqmp_virt_defconfig b/configs/xilinx_zynqmp_virt_defconfig
index cc93bc9b86..8a9703097e 100644
--- a/configs/xilinx_zynqmp_virt_defconfig
+++ b/configs/xilinx_zynqmp_virt_defconfig
@@ -5,7 +5,7 @@ CONFIG_TEXT_BASE=0x8000000
 CONFIG_SYS_MALLOC_LEN=0x4040000
 CONFIG_SYS_MALLOC_F_LEN=0x8000
 CONFIG_DM_GPIO=y
-CONFIG_DEFAULT_DEVICE_TREE="zynqmp-zcu100-revC"
+CONFIG_DEFAULT_DEVICE_TREE="zynqmp-zcu102-rev1.0"
 CONFIG_SPL_STACK_R_ADDR=0x18000000
 CONFIG_SPL_SIZE_LIMIT=0x2a000
 CONFIG_SPL_SIZE_LIMIT_PROVIDE_STACK=0x0
-- 
2.35.5

